packages/sit-onyx/src/styles/root.scss

```diff
@@ -2,8 +2,8 @@
 
 @layer onyx.utility {
   :root {
-    --onyx-font-family: "Source Sans 3 Variable", sans-serif;
-    --onyx-font-family-mono: "Source Code Pro Variable", monospace;
+    --onyx-font-family: var(--onyx-font-family-paragraph), sans-serif;
+    --onyx-font-family-mono: var(--onyx-font-family-mono), monospace;
```

Comment:
General concern: I am not sure if we can cover the font families with Figma variables since the name of the font family must be the exact same.

For example if the font family in Figma is "Source Sans 3" but technically its "Source Sans 3 Variable" this will not work.

Because currently, the font family is not applied correctly